install_config/install/advanced_install.adoc

```diff
@@ -1690,6 +1786,80 @@ ifdef::openshift-origin[]
 endif::[]
 ----
 
+[discrete]
+[[running-the-advanced-installation-tsb]]
+==== Optional: Deploying the Template Service Broker
+
+If you have xref:enabling-service-catalog[enabled the service catalog] and
+want to deploy the xref:configuring-template-service-broker[template service
+broker], run the following manual steps after the cluster installation
+completes:
+
+[WARNING]
+====
+The template service broker (TSB) is currently a Technology Preview feature and
+should not be used in production clusters. Enabling the TSB in this Technology
+Preview period currently requires opening unauthenticated access to the cluster.
+====
+
+. Run the following command:
++
+----
+<insert command from oc cluster up --service-catalog that I can't get to work>
+----
```

Comment:
if you're on the 3.7 stream, it's currently broken there again.

anyway the command is this:
`oc adm policy add-cluster-role-to-group system:openshift:templateservicebroker-client system:unauthenticated system:authenticated`

and it must be run as cluster admin.